pandas/io/pytables.py

```diff
@@ -423,6 +423,7 @@ class HDFStore(StringMixin):
     complevel : int, 0-9, default None
             Specifies a compression level for data.
             A value of 0 disables compression.
+            None is translated to 0.
```

Comment:
Since `None` and `0` are essentially equivalent, can't we just say "A value of 0 or None disables compression"

Also outside of PR scope: I think at some point, we should just change the default to `0` ?

Reply:
I agree not to have a new line.
If we change the default to `0`, should we raise an exception when `None` is given?

Reply:
Let's save changing the default for another PR.

Reply:
@gfyoung PR updated as told.